src/containers/forms/PlanForm/helpers.ts

```diff
@@ -412,7 +412,7 @@ export function doesFieldHaveErrors(
   errors: Array<FormikErrors<PlanActivityFormFields | PlanJurisdictionFormFields> | undefined>
 ) {
   return (
-    errors && Object.entries(errors[index] || {}).filter(([key, val]) => key === field).length > 0
+    errors && Object.entries(errors[index] || {}).filter(([key, _]) => key === field).length > 0
```

Comment:
Why `_` here?

Reply:
When you remove the code here breaks, so I kept it.